EIPS/eip-1154.md

```diff
@@ -0,0 +1,94 @@
+---
+eip: 1154
+title: Oracle Interface
```

Comment:
Please add a discussions-to URL.

Reply:
I've pointed it at this PR. I hope that is okay, but LMK if the preferred process is to have the discussion elsewhere or something.

Reply:
This PR exists for just this change, not the EIP itself. Please use somewhere that can be used for ongoing discussion of the EIP - such as an Ethereum Magicians thread, or an issue in this repo.